src/nycdb/sql.py

```diff
@@ -5,8 +5,26 @@ def create_table(table_name, fields):
     return sql
 
 
-def insert(table_name, row):
-    fields = ', '.join(row.keys())
-    placeholders = ', '.join(["%({})s".format(k) for k in row.keys()])
-    sql = "INSERT INTO {table_name} ({fields}) VALUES ({values});"
-    return sql.format(table_name=table_name, fields=fields, values=placeholders)
+def insert_many(table_name, rows):
+    '''
+    Given a table name and a list of dictionaries representing
+    rows, generate a (sql, template) tuple of strings that can be
+    passed to psycopg2.extras.execute_values() [1] to bulk insert all the
+    values for improved efficiency [2].
+
+    For example:
+
+        >>> insert_many('boop', [{'foo': 1, 'bar': 2}])
+        ('INSERT INTO boop (foo, bar) VALUES %s', '(%(foo)s, %(bar)s)')
+
```

Comment:
I think this example improves the readability of the code, and to make sure it's valid I enabled doctesting in pytest... hope that's ok, I can remove it and/or the doctesting if you want.